proto/viam/app/v1/app.proto

```diff
@@ -843,11 +856,16 @@ enum FragmentErrorType {
   FRAGMENT_ERROR_TYPE_CYCLE_DETECTED = 4;
 }
 
+message FragmentUsageStats {
+  int32 organizations = 1;
+  int32 machines = 2;
+  int32 machines_in_current_org = 3;
+}
+
 message FragmentUsage {
   string fragment_id = 1;
-  int32 organizations = 2;
-  int32 machines = 3;
-  int32 machines_in_current_org = 4;
```

Comment:
I forgot this was breaking. But I think we can handle that. I think its better than keeping these fields AND having a separate object for versions